pom.xml

```diff
@@ -109,6 +109,11 @@
         <artifactId>kryo-shaded</artifactId>
         <version>4.0.0</version>
       </dependency>
+      <dependency>
+        <groupId>io.grpc</groupId>
+        <artifactId>grpc-context</artifactId>
+        <version>1.12.0</version>
```

Comment:
Internally we use 1.4.0 due to compatibility issue. If 1.4.0 can work in this case, we might use that.